### PR TITLE
Use SymfonyStyle to output text in console

### PR DESCRIPTION
**Pull request by emodric**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | N/A

#### Summary

This adds a bit of style to console commands. Main visual differences are whitespaces, with occasional change in coloring.

I can add some example screenshots with before and after if needed.

Still needs tests.

### CODE REVIEW

**Comment by goetas** [comment]
Thanks a lot for this contribution. This is something I would like to have for 3.0! I think that screenshots are not needed.

Fixing tests for sure is necessary.  :)

**Comment by emodric** [comment]
@goetas I have updated the tests. I did have to increase the terminal width on Travis for the tests to pass since SymfonyStyle obviously wraps the output if the display is too small, causing output assertions to fail.

File: lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php [hunk 1]
```diff
@@ -25,6 +26,9 @@ abstract class DoctrineCommand extends Command
     /** @var DependencyFactory|null */
     private $dependencyFactory;
 
+    /** @var StyleInterface */
+    protected $io;
```
Comment (stof) [inline lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php:30]: should be private

Comment (emodric) [reply lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php:30]: Can't be private. It is used by other commands which extend from `DoctrineCommand`.

**Comment by goetas** [comment]
Thanks

**Pull request merged**